build/debian-base/Dockerfile.build

```diff
@@ -44,7 +44,6 @@ RUN echo "Yes, do as I say!" | apt-get purge \
     debconf-i18n \
     e2fslibs \
     e2fsprogs \
-    gcc-4.8-base \
```

Comment:
should be gcc-6-base

Reply:
added gcc-6-base

Reply:
Did you forget to push it?

Reply:
I took out gcc-6-base some time ago, because there were errors later in the build.

https://github.com/kubernetes/kubernetes/pull/52744#issuecomment-333691015